kibbeh/src/modules/search/SearchBarController.tsx

```diff
@@ -39,6 +39,13 @@ export const SearchBarController: React.FC<SearchControllerProps> = ({}) => {
     [text]
   );
   const { push } = useRouter();
+  const [results, setResults] = useState<Array<Room | User>>([]);
+
+  useEffect(() => {
+    if (data?.rooms && data?.users) {
+      setResults([...data.rooms, ...data.users]);
+    }
+  }, [data]);
```

Comment:
don't useEffect/useState, just stick it in a variable

Reply:
but data comes from a hook right, so i need to use useEffect/useState

Reply:
ok nvm, i removed them